src/db/group.rs

```diff
@@ -17,20 +17,44 @@ pub struct Group {
     /// The name of the group
     pub name: String,
 
+    /// Notes for the group
+    pub notes: Option<String>,
+
+    /// ID of the group's icon
+    pub icon_id: Option<usize>,
+
+    /// UUID for a custom group icon
+    pub custom_icon_uuid: Option<String>,
+
     /// The list of child nodes (Groups or Entries)
     pub children: Vec<Node>,
 
     /// The list of time fields for this group
     pub times: Times,
+
+    /// Whether the group is expanded in the user interface
+    pub is_expanded: bool,
+
+    /// Default autotype sequence
+    pub default_autotype_sequence: Option<String>,
+
+    /// Whether autotype is enabled
+    // TODO: in example XML files, this is "null" - what should the type be?
+    pub enable_autotype: Option<String>,
+
+    /// Whether searching is enabled
+    // TODO: in example XML files, this is "null" - what should the type be?
+    pub enable_searching: Option<String>,
+
+    pub last_top_visible_entry: Option<String>,
```

Comment:
@sseemayer I'm guessing this is an entry's UUID? We could probably document that.

Reply:
That item was previously intentionally left undocumented because I had no clue what "last top visible" is supposed to mean. It is the UUID of some entry, but even going back into the original KeePass sourcecode, it is not 100% clear to me what it is about - something about restoring selected item states when re-opening a database. 

I will add some documentation text that clarifies that it is supposed to be an item UUID, but won't bother to reverse engineer this field further for the time being.